picamctl/picamctl.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```

Comment:
#!/bin/sh or #!/usr/bin/env bash are better for platform compatibility.

Reply:
Ok, I will check this out.

Reply:
The script has been entirely refactored for sh.